web/controllers/post_controller.ex

```diff
@@ -25,7 +25,7 @@ defmodule Healthlocker.PostController do
   end
 
   def index(conn, _params) do
-    posts = Repo.all(Post)
+    posts = Repo.all(from p in Post, where: ilike(p.content, "%#story%"))
```

Comment:
I did most of the controller tests last night, will finish them up this morning.